cicd/builds/running-entitled-builds.adoc

```diff
@@ -21,9 +21,6 @@ include::modules/builds-source-input-satellite-config.adoc[leveloffset=+2]
 
 include::modules/builds-strategy-docker-entitled-satellite.adoc[leveloffset=+2]
 
-include::modules/builds-strategy-docker-squash-layers.adoc[leveloffset=+1]
-
-
```

Comment:
I agree that this link should be removed from its current position.

That said, I think a pointer to https://docs.openshift.com/container-platform/4.7/cicd/builds/build-strategies.html#builds-strategy-docker-build_build-strategies (please translate that literal 4.7 link to the appropriate module) should be included in the modules 
- modules/builds-strategy-docker-entitled-subman.adoc
- modules/builds-strategy-docker-entitled-satellite.adoc

so the user has a reminder of all the docker build strategy options available, including squash layers

to be clear, do not re-include all that content ... just a relative link to where the content currently lives

Reply:
I've placed the cross-reference you suggested in the nearest "Additional resources" section we're allowed to use.